js/node/node-backend.js

```diff
@@ -26,59 +26,7 @@ if (!isMainThread) {
 }
 
 // state machine to enable calling multiple functions that need a thread pool at once
-let state = 'none'; // 'initializing', 'running', 'exiting'
```

Comment:
this logic was lifted into `workers.ts` in o1js, see https://github.com/o1-labs/o1js/pull/1931